examples/panda_example.py

```diff
@@ -1,33 +1,64 @@
 import sys
+import os
 import numpy as np
-import gym
-import urdfenvs.panda_reacher
-from MotionPlanningEnv.sphereObstacle import SphereObstacle
-from MotionPlanningGoal.staticSubGoal import StaticSubGoal
+import gymnasium as gym
+# import urdfenvs.panda_reacher
+# from MotionPlanningEnv.sphereObstacle import SphereObstacle
+# from MotionPlanningGoal.staticSubGoal import StaticSubGoal
```

Comment:
These lines can be removed instead of commented.

Reply:
Done